templates/agent/scripts/couchbase.py.j2

```diff
@@ -0,0 +1,158 @@
+import json
+import urlparse
+import requests
+import sys
+
+
+# Usage examples:
+# Cluster:
+
+# python couchbase.py cluster maxBucketCount int
+# python couchbase.py cluster name string
+# python couchbase.py cluster buckets.uri string
+# python couchbase.py cluster storageTotals.hdd.free int
+# python couchbase.py cluster nodes string_list
+
+# Bucket
+
+# python couchbase.py bucket names string_list
+# python couchbase.py bucket <name>  stats.op.samples.ops avg
+
+# Node
+# python couchbase.py node <hostname> interestingStats.couch_views_data_size int
+
+
+couchbase_api_endpoint = "{{ zabbix_couchbase.api_endpoint }}"
```

Comment:
As discussed - we may want the script to be static and kept as is - driving the changes from a config file. Consider removing the `{{ }}` and use defaults localhost.

Depending how much work it is, consider adding the support for a config file.